packages/pages/src/components/analytics/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from "./interfaces";
```

Comment:
Do you actually need to export all of these things or can just export the component?

Reply:
I'm not 100% sure on where we should make the package boundaries. I could see use cases for exposing these.  I was also somewhat following slapshot's conventions for testing but their tests are setup differently.

Reply:
I would only export the specific things you'd want a user to be able to import.

Reply:
ok, i've removed the interfaces.